src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorTimer.kt

```diff
@@ -64,13 +73,20 @@ class GardenVisitorTimer {
         }
 
         if (queueFull) {
-            if (sixthVisitorArrivalTime != 0L && visitorJustArrived) {
+            if (visitorJustArrived && visitorsAmount - lastVisitors == 1) {
```

Comment:
stops the 6th visitor counter being set to 12 minutes on entering the garden for the first time since joining hypixel